openapi-spec/swagger.yaml

```diff
@@ -1446,27 +1396,32 @@ definitions:
             readOnly: true
           name:
             type: string
-            example: volumeGroup-demo
           description:
             type: string
-            example: volume group test
           availabilityZone:
             type: string
-            example: default
             readOnly: true
           status:
             type: string
             readOnly: true
           poolId:
             type: string
             readOnly: true
-          profiles:
+          AddVolumes:
+            type: array
+            items: 
+              type: string
+            readOnly: true
+          RemoveVolumes:
             type: array
             items: 
               type: string
-            example: 
-              - 993c87dc-1928-498b-9767-9da8f901d6ce 
-              - 90d667f0-e9a9-427c-8a7f-cc714217c7bd
+            readOnly: true
+          GroupSnapshots:
```

Comment:
GroupSnapshots have not been implemented yet.  Why are they added now?

Reply:
OK